### PR TITLE
Feature/role

### PR DESCRIPTION
**Pull request by Durgesh00**
Ticket:- https://trello.com/c/oHfxyQK3/162-when-invitation-sent-to-user-role-value-in-state-is-roleid-should-be-title

### CODE REVIEW

File: src/modules/admin/inviteUser/InviteUserContainer/apis.js [hunk 1]
```diff
@@ -37,3 +37,18 @@ export const getUsersApi = (organization_id) => {
...
   });
 };
+
+export const getUserRole = () => {
+  return axios({
+    method: 'get',
+    url: `${SERVER_URL}admin/roles`,
+    headers: {
+      'Content-Type': 'application/json',
+      'access-token': local.getItem('accessToken'),
```
Comment (pooja-mane) [inline src/modules/admin/inviteUser/InviteUserContainer/apis.js:47]: Not in this PR.
In another task, Can we write common method to get headers for api calls as code refactoring task.

Comment (somnath77) [reply src/modules/admin/inviteUser/InviteUserContainer/apis.js:47]: get method defined in apiHelper can be used instead of defining new axios method here

**Review by somnath77 — APPROVED** [review]

File: src/modules/admin/inviteUser/InviteUserContainer/index.js [hunk 2]
```diff
@@ -48,12 +56,17 @@ const InviteUserContainer = () => {
 
   const handleSendInvitation = async () => {
     const { email, role } = usersData;
-
+    const roleValue = role.value;
+    const roleLabel = role.label;
```
Comment (pooja-mane) [inline src/modules/admin/inviteUser/InviteUserContainer/index.js:60]: can we rename `roleLabel` to `raleName` and `roleValue` to 'roleId' ?

**Review by pooja-mane — APPROVED** [review]
Approved with minor comment

**Review by somnath77 — APPROVED** [review]

**Pull request merged**